shell/components/nav/Header.vue

```diff
@@ -113,6 +115,12 @@ export default {
       return getProduct();
     },
 
+    vendor() {
+      this.$store.getters['management/all'](MANAGEMENT.SETTING)?.find((setting) => setting.id === 'ui-pl');
```

Comment:
Looks like the import is missing based on the github warning.

Reply:
It should be fixed now.